app/views/dashboard/_status_links.html.erb

```diff
@@ -1,3 +1,4 @@
+<!-- TODO: Replace with either just out of sync or origin specific interval -->
```

Comment:
I think this needs to use the same logic that looks at all relevant origins, you should e.g. see hosts with active report from puppet or ansible with respective intervals